planemo/workflow_lint.py

```diff
@@ -212,6 +213,18 @@ def check_json_for_untyped_params(j):
     # creator
     if not len(workflow_dict.get("creator", [])) > 0:
         lint_context.warn("Workflow does not specify a creator.")
+    else:
+        creators = workflow_dict.get("creator")
```

Comment:
Technically the the creators variable could be `None` if there is no "creator" in your workflow, in which case you might run into an error on line 221, calling `.get` on a none-type variable.

```suggestion
        creators = workflow_dict.get("creator", [])
```

Reply:
Actually it might be a good idea to do `creators = workflow_dict.get("creator", [])` on line 213 and refer to that.